src/interface/src/app/treatments/map-rectangle/map-rectangle.component.html

```diff
@@ -0,0 +1,19 @@
+<mgl-geojson-source id="rectangle-source">
+  <mgl-feature [geometry]="rectangleGeometry"></mgl-feature>
+</mgl-geojson-source>
+<mgl-layer
+  id="selection-rectangle"
+  type="line"
+  [source]="sourceName"
+  [paint]="{
+    'line-color': '#007dff',
```

Comment:
I don't know if it's worth the effort yet, but if there's a way to store these styles as constants somewhere, maybe that would help us as this grows?

Reply:
Yes i think that's probably the next step once we have more layers